marlowe-playground-client/src/JavascriptEditor/State.purs

```diff
@@ -85,8 +116,85 @@ handleAction _ SendResultToBlockly = do
 editorResize :: forall state action msg m. HalogenM state action ChildSlots msg m Unit
 editorResize = void $ query _jsEditorSlot unit (Monaco.Resize unit)
 
-editorSetValue :: forall state action msg m. String -> HalogenM state action ChildSlots msg m Unit
-editorSetValue contents = void $ query _jsEditorSlot unit (Monaco.SetText contents unit)
+decorationHeader :: Array String
+decorationHeader =
```

Comment:
instead of these arrays you can use purescripts multiline strings
```purescript
c = """
some
string
"""
```
and then use `lines` if you need the number of lines.